modules/installation-special-config-storage.adoc

```diff
@@ -18,19 +35,35 @@ This feature:
 * Requires no user intervention for providing passphrases
 * Uses AES-256-XTS encryption, or AES-256-CBC if FIPS mode is enabled
 
-There are two different supported encryption modes:
+[id="installation-special-config-encryption-threshold_{context}"]
+=== Configuring an encryption threshold
+
+In {product-title}, you can specify a requirement for more than one Tang server. You can also configure the TPM v2 and Tang encryption modes simultaneously, so that the boot disk data can be decrypted only if the TPM secure cryptoprocessor is present and the Tang servers can be accessed over a secure network. 
 
-* TPM v2: This is the preferred mode. TPM v2 stores passphrases in a secure cryptoprocessor.
-* Tang: To use Tang to encrypt your cluster, you need to use a Tang server. Clevis implements decryption on the client side.
+You can use the `threshold:` attribute in your Butane configuration to define the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur. The threshold is met when the stated value is reached through any combination of the declared conditions. For example, the `threshold:` value of `2` in the following configuration can be reached by accessing the two Tang servers, or by accessing the TPM secure cryptoprocessor and one of the Tang servers:
+
```

Comment:
Do you want a _.Example Butane configuration_ here? 
Should you have an example for accessing a cryptoprocessor?

Reply:
That's a good idea, I will add that heading. The `tpm2: true` enables the TPM v2 encryption mode, which is the mode that uses the secure cryptoprocessor in the server. My understanding is that if the processor is enabled in the BIOS (as per step 1 of the procedure later in this module), then this setting is all that is required.

I will add some call outs to this example that help clarify that.

Reply:
Resolved.